src/Microsoft.DotNet.TestFramework/Microsoft.DotNet.TestFramework.TestInstance.cs

```diff
@@ -105,8 +107,13 @@ public TestInstance WithBuildArtifacts()
                                  .Where(file =>
                                  {
                                      file = file.ToLower();
-                                     return file.Contains($"{Path.DirectorySeparatorChar}bin{Path.DirectorySeparatorChar}") 
+
+                                     var isArtefact = file.Contains($"{Path.DirectorySeparatorChar}bin{Path.DirectorySeparatorChar}") 
```

Comment:
http://grammarist.com/spelling/artefact-artifact/

I'll switch to artifact, since the CLI is a US product :)